### DISCUSSION TITLE
[A-L] (2023/24) Foglio 3 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#182] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-10-22 204412](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/3602f3f1-433e-46fe-adce-ce4c1a182054)

**Comment by Elia-Belli** [comment, marked as answer]
Innanzitutto controlliamo se questa eq. ha soluzione, ovvero se $(a,n)|b$:
$(a,n)=(121,33)=11$ e osserviamo che $11|22$, allora esiste almeno una soluzione.

Divido tutto per $(a,n)=11$ in modo da avere un'eq. equivalente più semplice da risolvere:
$121X\equiv 22(33)\iff \frac{121}{11}X\equiv \frac{22}{11}(\frac{33}{11})=11X=2(3)$
Cerco un inverso di $11$ in $\mathbb{Z}_3$, ovvero $x: 11x=1(3)$, ad esempio $2$ infatti $22=1(3)\rightarrow 22-1=3\cdot 7$
Moltiplico emtrambi i lati per $[2]=[11]^{-1}$, ottenendo $[2][11]X=[4]\rightarrow X=[4]$ in $\mathbb{Z}_3$, allora $x_0=1$ è soluzione.

Tutte le soluzioni sono $\\{1,4,7,10,13,16,19,22,25,28,31\\}$

**Comment by daromacarlo** [comment]
![Immagine 2023-12-12 104958](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/a36db155-0eb4-4c7d-894e-f0a28fc004df)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/a3cc6c30-f772-4144-9d97-0e0dccd7e6ba)
